ansible/playbooks/master.yml

```diff
@@ -6,9 +6,15 @@
     - ../vars.yml
 
   roles:
-    - keepalived
-    - haproxy
-    - masters
-    - { role: initial-master, when: "groups['masters'][0] == inventory_hostname" }
-    - { role: cni, when: "groups['masters'][0] == inventory_hostname" }
-    - { role: other-masters, when: "groups['masters'][0] != inventory_hostname" }
+    - ../roles/keepalived
+    - ../roles/haproxy
+    - ../roles/masters
+    - {
+        role: ../roles/initial-master,
+        when: "groups['masters'][0] == inventory_hostname",
+      }
+    - { role: ../roles/cni, when: "groups['masters'][0] == inventory_hostname" }
+    - {
+        role: ../roles/other-masters,
+        when: "groups['masters'][0] != inventory_hostname",
+      }
```

Comment:
Strange it puts multiline for the 1st and 3rd but not the second, anything we can tweak to make it consistent?

Reply:
What a weird convention, looks to be line length dependent. I'll see what I can do